lib/sidekiq/cron/job.rb

```diff
@@ -464,19 +475,18 @@ def save
         return false unless valid?
 
         Sidekiq.redis do |conn|
-
           # Add to set of all jobs
-          conn.sadd self.class.jobs_key, [redis_key]
+          conn.sadd self.class.jobs_key(@namespace), [redis_key]
 
           # Add informations for this job!
-          conn.hset redis_key, to_hash.transform_values! { |v| v || "" }
+          conn.hset redis_key, (to_hash.transform_values! { |v| v || '' })
```

Comment:
This line doesn't need to be changed

Reply:
Well that line is marked with the following from Rubocop:

> Lint/AmbiguousBlockAssociation - Parenthesize the param `to_hash.transform_values! { |v| v || '' }` to make sure that the block will be associated with the `to_hash.transform_values!` method call.

Whick looked important enough to me to change it, but I will revert it.